kong/core/handler.lua

```diff
@@ -136,6 +137,17 @@ return {
     -- Only executed if the `router` module found an API and allows nginx to proxy it.
     after = function()
       local ctx = ngx.ctx
+      local var = ngx.var
+
+      local uri      = ctx.uri
+      local uri_args = var.args
+
+      if uri_args then
+        uri = uri .. "?" .. uri_args
+      end
```

Comment:
rather than the string concatentation here, how do we feel about defining the proxy_pass directive as

`proxy_pass $upstream_scheme://kong_upstream$upstream_url$is_args$args;`

i dont have strong feels one way or another, just wondering about idiomatic approach and performance (ngx.var access)